docs/SNAPTELD_CONFIGURATION.md

```diff
@@ -17,11 +17,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# snapd Configuration File
+# snapteld Configuration File
 
-snapd supports being configured through a configuration file located at a default location of `/etc/snap/snapd.conf` on Linux systems or by passing a configuration file in through the `--config` command line flag when starting snapd. YAML and JSON are currently supported for configuration file types.
+snapteld supports being configured through a configuration file located at a default location of `/etc/snap/snapteld.conf` on Linux systems or by passing a configuration file in through the `--config` command line flag when starting snapteld. YAML and JSON are currently supported for configuration file types.
```

Comment:
I think we can keep it /etc/snap since there's no conflicts. This will allow keyrings file to be kept in same location as well.